.travis.yml

```diff
@@ -112,7 +124,12 @@ matrix:
     env: BEAKER_set="ubuntu-1804" BEAKER_PUPPET_COLLECTION=puppet6
     bundler_args:
     script: bundle exec rake beaker
-
+  allow_failures:
+  # waiting for centos 8 release
```

Comment:
These should work if we're going to support it.

Reply:
except centos8 hasnt been released yet.
I was just copying your example.
Do you want me to try and change it to rhel8?

Reply:
Yes, please. We want to ensure these tests are passing.